infra/experimental/SystemSan/SystemSan.cpp

```diff
@@ -290,6 +292,7 @@ void inspect_for_arbitrary_file_open(pid_t pid, const user_regs_struct &regs) {
     struct stat dirstat;
     if (stat(path_absolute_topdir.c_str(), &dirstat) != 0) {
       report_bug(kArbitraryFileOpenError);
+      std::cerr << "===File opened: " << path.c_str() << ", flags = " << regs.rdx << "===\n";
```

Comment:
Also can we refactor this logging into its own function? e.g. something called `log_file_open`

Reply:
Done